test/equilibrate_test_4.jl

```diff
@@ -0,0 +1,68 @@
+#Tests various bifurcation capabilities.
+for rn in reaction_networks_standard
+    p_vals = map(amp->amp*rand(length(rn.params)),[1.,10.,100.])
+    param1 = rand(rn.params)
+    param2 = rand(rn.params)
+    range_tupple = (1.,rand(5.:10.))
+    range1 = 1.: rand(5.:10.)
+    range2 = 1.: rand(5.:10.)
+    for p in p_vals
+        bif1 = bifurcations(rn,p,param1,range_tupple)
+        bif2 = bifurcations(SimpleHCBifurcationSolver(), rn,p,param1,range_tupple)
+        bif_grid = bifurcations_grid(rn,p,param1,range1)
+        bif_grid_2d = bifurcations_grid_2d(rn,p,param1,range1,param2,range2)
```

Comment:
why so many API functions? Can this not just work by dispatch?

Reply:
@korsbo suggested a method were one gave a network topology type to designate which one to use, in the same spirit the solvers are given. I thought this would generate some messy since each method have requires different types of input

We have:
bifurcations requires `AbstractReactionNetwork`, `Vector{Float64}`, `Symbol`, `Tuple{Number,Number}` (technically the last one can be anything, but something like this which contains two numbers)
bifurcation_grid requires `AbstractReactionNetwork`, `Vector{Float64}`, `Symbol`, `AbstractRange`
bifurcation_grid_2d requires `AbstractReactionNetwork`, `Vector{Float64}`, `Symbol`, `AbstractRange`, `Symbol`, `AbstractRange`
bifurcation_diagram_grid requires `AbstractReactionNetwork`, `Vector{Float64}`, `Symbol`, `AbstractRange`, `Symbol`, `Tuple{Number,Number}`

I thought it would be clearer to have a separate function name for each, but it would be possible, and maybe a bit simpler, to have a single function , `bifurcations`, and depending on what form the input have it can have a dispatch for the corresponding type of diagram.